mlir/utils/performance/parameterSweeps.py

```diff
@@ -288,8 +300,12 @@ async def dropGoodConfig(config, options: Options, paths: Paths):
         if isinstance(config, MLIROnlyConfig):
             print(f"{result.name}: {config!r}")
         else:
+            print("" + "-" * 100)
```

Comment:
```suggestion
            print("-" * 100)
```

This does the same thing. 

```python3
>>> print("" + "-" * 100)
----------------------------------------------------------------------------------------------------
>>> print("-" * 100)
----------------------------------------------------------------------------------------------------
```